CalibTracker/SiStripESProducers/plugins/real/SiStripQualityESProducer.cc

```diff
@@ -36,7 +36,7 @@ SiStripQualityESProducer::SiStripQualityESProducer(const edm::ParameterSet& iCon
 
 std::shared_ptr<SiStripQuality> SiStripQualityESProducer::produce(const SiStripQualityRcd& iRecord)
 {
-  std::shared_ptr<SiStripQuality>  quality{new SiStripQuality};  
+  std::shared_ptr<SiStripQuality>  quality = std::make_shared<SiStripQuality>();  
```

Comment:
@gartung the comment on `auto` applies here too

Reply:
@ghellwig There are many instances in this package where auto can be used. Should I change all of them in one commit?

Reply:
only those, that you are touching anyway